### PR TITLE
PXB-3126 Add quickstart for PXB 8.0

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/quickstart-docker.md
	modified:   docs/quickstart-overview.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/draft.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Create a Docker container based on Percona XtraBackup image with the docker run command. If it is not the first time you use the selected image, Docker uses the image available locally.
+
+1. Create docker network
+
+    ```{.bash data-prompt="$"}
+    $ docker network create mynet
+    ```
+
+2. Create a Docker container based on Percona XtraBackup image with the docker run command.
```
Comment (patrickbirch) [inline docs/draft.md]: Should we add that tag to the container so that it matches the PS version? Otherwise, in a year, this quickstart may not work.

File: docs/draft.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    $ docker run  --name pxb80 --net=mynet --volumes-from psmysql percona/percona-xtrabackup:8.0.34 \
+    xtrabackup --port=3306 --backup --datadir=/var/lib/mysql --target-dir=/backup \
+    --user=root --password=secret
+    ```
+
+    With parameter name you give a meaningful name to your new Docker container so that you could easily locate it among your other containers.
+
+    The volumes-from flag refers to Percona Server for MySQL and indicates that you intend to use the same data as the Percona Server for MySQL container.
+
+3. Run the container with exactly the same parameters that were used when the container was created:
```
Comment (patrickbirch) [inline docs/draft.md]: Docker run was executed, now we have a Docker start? What's the difference? 
Don't start the item with "Run the container" since we just used "Docker run".

File: docs/quickstart-docker.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
+# Start Percona XtraBackup in a docker continer, take and prepare a backup
+
+As soon as you [start Percona Server in a docker container](https://docs.percona.com/percona-server/8.0/quickstart-docker.md) and [create a database and table in Percona Server](https://docs.percona.com/percona-server/8.0/quickstart-database.md), you are ready to backup your databes with Percona XtraBackup.
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: Tell me about this other task first.

File: docs/draft.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+## Pull the Percona XtraBackup image
+
```
Comment (patrickbirch) [inline docs/draft.md]: No overview topic?
What is the purpose of the topics?
What will happen during the quickstart?

File: docs/draft.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/draft.md]: Sorry, this was a draft doc, I've forgotten to remove it.

File: docs/draft.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/draft.md]: Sorry, this was a draft doc, I've forgotten to remove it.

File: docs/draft.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/draft.md]: Sorry, this was a draft doc, I've forgotten to remove it.

**Comment by satya-bodapati** [comment]
title is misleading. It's all about Docker.  Lets rename this section How to use xtrabackup with docker (or something similar)

File: docs/quickstart-docker.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: Added.

File: docs/quickstart-docker.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+    To use the `docker run` command, specify the name or ID of the image you want to use and, optionally, some flags and arguments that modify the container's behavior. 
+
+    The command has the following options:
+
+    | Option | Description |
+    |---|---|
+    | `--volumes-from` | Refers to Percona Server for MySQL and indicates that you intend to use the same data as the `psmysql` container.|
+    | `it` | Interacts with the container and be a pseudo-terminal. |
+    | `--user root` | Sets the user to root inside the container. This option is required to access the MySQL data directory and run the xtrabackup command. |
```
Comment (satya-bodapati) [inline docs/quickstart-docker.md]: keep the --rm option documented (we are using it elsewhere)

File: docs/quickstart-docker.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+    For this document, we are using the 8.0.34 tag. In Docker, a tag is a label assigned to an image. Tags are used to maintain different versions of an image. If we did not add a tag, Docker uses latest as the default tag and downloads the latest image from [percona/percona-xtrabackup on the Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup).
+
+    * Run a Docker container example
+
+        ```{.bash data-prompt="$"}
+        $ docker run --volumes-from psmysql -v backupvol:/backup_8034 -it --user root percona/percona-xtrabackup:8.0.34 /bin/bash -c "xtrabackup --backup --datadir=/var/lib/mysql/ --target-dir=/backup_8034 --user=root --password; xtrabackup --prepare --target-dir=/backup_8034"
+        ```
+
+        You are prompted to enter the password, which is `secret`, in our example.
```
Comment (satya-bodapati) [inline docs/quickstart-docker.md:51]: we can add a note here probably

For convenience you can add the password=secret to command but be aware that it is visible in docker ps, docker ps -a (docker history) and regular ps command.

Comment (satya-bodapati) [reply docs/quickstart-docker.md:51]: not exactly here. somewhere around

File: docs/quickstart-docker.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+
+        2023-12-14T15:45:20.614336-00:00 0 [Note] [MY-012980] [InnoDB] Shutdown completed; log sequence number 31467030 
+
+        2023-12-14T15:45:20.619892-00:00 0 [Note] [MY-011825] [Xtrabackup] completed OK! 
+        ```
+
+    This command runs a Docker container from the `percona/percona-xtrabackup:8.0.34` image and mounts two volumes: one from another container named `psmysql`, which contains Percona Server data directory, and another named `backupvol`, which is where the backup files are stored. The command also sets the user to root and passes the root password for MySQL as an environment variable. The command then executes two steps: 
+    
+     * Runs `xtrabackup` with the `--backup` option to copy the data files from `/var/lib/mysql/` to `/backup_8034`
+
```
Comment (satya-bodapati) [inline docs/quickstart-docker.md:85]: somwhere around this, you can probably mention that the logs from xtrabackup can be seen with the command
` sudo docker container logs <container-name>`

Hmm, that reminds me did we name the container with --name option?

**Review by satya-bodapati — CHANGES_REQUESTED** [review]
I would love to see few more "Advanced topics with docker". These do not fall under 'quickstart". Separate PRs
1. incremental backups with docker
2. backups to cloud/restore from cloud
3. using pxb docker image to take backup and restore a server that is not on docker

**Comment by alina-derkach-oaza** [comment]
> I would love to see few more "Advanced topics with docker". These do not fall under 'quickstart". Separate PRs

Sure, I'll create separate Jiras, to add these topics

File: docs/quickstart-docker.md (hunk 5 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: We do not use --rm option in this very document, so I'll add the --rm description to quickstart-restore-backup.md

File: docs/quickstart-docker.md (hunk 7 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md:85]: Added the --name option to the command and to the option descriptions.

File: docs/quickstart-docker.md (hunk 6 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md:51]: added to the --password option description.

File: docs/quickstart-docker.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
+# Start a docker container and take a backup
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: Docker should be capitalized.

File: docs/quickstart-docker.md [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+
+Percona XtraBackup works in combination with a database server, Percona Server, in our scenario. To use Percona XtraBackup, you need to run it in its own Docker container and connect it to the Percona Server container. As soon as you [start Percona Server in a docker container](https://docs.percona.com/percona-server/8.0/quickstart-docker.html) and [create a database and table in Percona Server](https://docs.percona.com/percona-server/8.0/quickstart-database.html), you are ready to backup your databes with Percona XtraBackup.
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: Percona Server for MySQL

Comment (patrickbirch) [inline docs/quickstart-docker.md]: works in combination with a MySQL-compatible database

File: docs/quickstart-docker.md [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+
+The following steps create a Docker volume, start Percona XtraBackup in a docker continer, take and prepare a backup of Percona Server for MySQL database.
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: Maybe change this to an order list?

File: docs/quickstart-docker.md [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+    | `backupvol` | Indicates the persistent storage for the database. |
+    | `psmysql`    | Indicates the Percona Server for MySQL container.  |
+    |`--password`   | Prompts the user to enter the password for the root user. For convenience you can add the password for the root user to this option, for example, `--password=secret`. Then the password will be passed automaticaly when running the command. Note that if you specify the password with `--password=secret`, the password is visible in `docker ps`, `docker ps -a` (docker history) and regular ps command.          |
+    | `8.0.34` | Use this tag to specify a specific version. |
+
+    When running Percona XtraBackup from a container and connecting to a Percona Server container, we recommend using the `–-user root` option in the Docker command.
+
+    For this document, we are using the 8.0.34 tag. In Docker, a tag is a label assigned to an image. Tags are used to maintain different versions of an image. If we did not add a tag, Docker uses latest as the default tag and downloads the latest image from [percona/percona-xtrabackup on the Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup).
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: Using tags also ensure that we are using compatible versions of PS for MySQL and PXB.

File: docs/quickstart-docker.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+        2023-12-21T16:16:10.796716-00:00 0 [Note] [MY-011825] [Xtrabackup] recognized server arguments: --datadir=/var/lib/mysql/2023-12-21T16:16:10.799979-00:00 0 [Note] [MY-011825] [Xtrabackup] recognized client arguments: --backup=1 --target-dir=/backup_8034 --user=root --password
+        
+        Enter password:
+        ```
+    
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+
+        2023-12-14T15:45:16.277503-00:00 0 [Note] [MY-011825] [Xtrabackup] recognized server arguments: --datadir=/var/lib/mysql/
```
Comment (patrickbirch) [inline docs/quickstart-docker.md:65]: we may be able to see the first and last section and use an ellipses instead of the entire backup

File: docs/quickstart-restore-backup.md [hunk 13]
```diff
@@ -0,0 +1,258 @@
...
+
+4. Remove all created files in `myvol2` to allow xtrabackup restore data to `myvol2`
+
+    ```{.bash data-prompt="$"}
+    $ docker run --volumes-from psmysql2 -v backupvol:/backup_8034 -it --rm --user root percona/percona-xtrabackup:8.0.34 /bin/bash -c "rm -rf /var/lib/mysql/*"
+    ```
+
+    If the command executes successfully, the expected output is empty.
+
+    The `--rm` option automatically removes the temporary container created from percona/percona-xtrabackup:8.0.34 image after the container exits.
```
Comment (patrickbirch) [inline docs/quickstart-restore-backup.md]: Move this information before the command is run.

Comment (alina-derkach-oaza) [reply docs/quickstart-restore-backup.md]: done

File: docs/quickstart-docker.md (hunk 12 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md:65]: Added

File: docs/quickstart-docker.md (hunk 11 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: Added

File: docs/quickstart-docker.md (hunk 10 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: added a list

File: docs/quickstart-docker.md (hunk 9 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: fixed

Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: fixed

File: docs/quickstart-docker.md (hunk 8 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: fixed

File: docs/quickstart-docker.md (hunk 10 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: Tried it. It does not look good.

File: docs/quickstart-overview.md [hunk 14]
```diff
@@ -1,35 +1,48 @@
...
+# Overview
 
-Percona XtraBackup (PXB) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+Percona XtraBackup (PXB) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems.
 
-## Install Percona XtraBackup:
+To start with Percona XtraBackup quickly, we recommend using Docker and this Quickstart guide focuses on this installation method. You can explore alternative installation options in the [Install](installation.md) section.
 
-You can install Percona XtraBackup using different methods:
+### Why to start working with Percona XtraBackup in a Docker container
```
Comment (patrickbirch) [inline docs/quickstart-overview.md]: Why to start?

File: docs/quickstart-overview.md [hunk 15]
```diff
@@ -1,35 +1,48 @@
...
 
-* [Use the Percona Repositories](installation.md)
+With Docker, you can build, deploy, run, update, and manage containers, which isolate applications from the host system. Docker containers are built from Docker images, which are snapshots of the configuration needed to run an applications, such as the code, and libraries. You can run Percona XtraBackup in a Docker container without installing the product.
```
Comment (patrickbirch) [inline docs/quickstart-overview.md]: The paragraph starts talking about containers, then images, and then back to containers.

Maybe it would flow better with:
Images
containers and then Percona XtraBackup (why did you add the PXB abbreviation if you are not using it?)
The last sentence may be something like:
A Docker container lets you work with Percona XtraBackup without installing the product on a local drive.

File: docs/quickstart-overview.md [hunk 16]
```diff
@@ -1,35 +1,48 @@
...
 
-* [Use APT](apt-repo.md)
+## Purpose of the Quickstart
 
-* [Use YUM](yum-repo.md)
+This section provides you with the basic steps to quiclky start Percona XtraBackup in a docker continer, take a backup, prepare a backup and restore a backup of Percona Server for MySQL database.
```
Comment (patrickbirch) [inline docs/quickstart-overview.md]: Docker container

File: docs/quickstart-overview.md [hunk 17]
```diff
@@ -1,35 +1,48 @@
...
 
-## For high availability
+* [Start Percona Server in a docker container](https://docs.percona.com/percona-server/8.0/quickstart-docker.html)
+* [Create a database and table in Percona Server](https://docs.percona.com/percona-server/8.0/quickstart-database.html)
 
-Percona XtraDB Cluster (PXC) is a 100% open source, enterprise-grade, highly available clustering solution for MySQL multi-master setups based on Galera. PXC helps enterprises minimize unexpected downtime and data loss, reduce costs, and improve performance and scalability of your database environments supporting your critical business applications in the most demanding public, private, and hybrid cloud environments. 
+### Supported storage engines
 
-Install [Percona XtraDB Cluster](https://docs.percona.com/percona-xtradb-cluster/8.0/install/index.html).
+Percona XtraBackup 8.0 can back up data from InnoDB, XtraDB, MyISAM, and MyRocks tables on MySQL servers as well as Percona Server for MySQL with XtraDB, Percona Server for MySQL, and Percona XtraDB Cluster.
```
Comment (patrickbirch) [inline docs/quickstart-overview.md]: Percona Server is mentioned twice.
Storage engines are InnoDB, XtraDB, MyISAM, and MyRocks.
The servers should be listed in a separate paragraph.

File: docs/quickstart-overview.md [hunk 18]
```diff
@@ -1,35 +1,48 @@
...
 
-## For Monitoring and Management
+### Limitations
 
-Percona Monitoring and Management (PMM) monitors and provides actionable performance data for MySQL variants, including Percona Server for MySQL, Percona XtraDB Cluster, Oracle MySQL Community Edition, Oracle MySQL Enterprise Edition, and MariaDB. PMM captures metrics and data for the InnoDB, XtraDB, and MyRocks storage engines, and has specialized dashboards for specific engine details.
+Percona XtraBackup 8.0 does not support making backups of databases created in versions before 8.0 of MySQL, Percona Server for MySQL or Percona XtraDB Cluster.
```
Comment (patrickbirch) [inline docs/quickstart-overview.md]: or versions greater than 8.0, such as the Innovation releases, 8.1 and 8.2.

File: docs/quickstart-overview.md [hunk 19]
```diff
@@ -1,35 +1,48 @@
...
+
+## Steps to perform
+
+In this section, you will learn how to:
+
+* [Start a docker container and take a backup](quickstart-docker.md)
```
Comment (patrickbirch) [inline docs/quickstart-overview.md]: Should be an ordered list

File: docs/quickstart-docker.md [hunk 20]
```diff
@@ -0,0 +1,143 @@
+# Start a Docker container and take a backup
+
+Percona XtraBackup works in combination with a MySQL-compatible database, Percona Server for MySQL, in our scenario. To use Percona XtraBackup, you need to run it in its own Docker container and connect it to the Percona Server for MySQL container. As soon as you [start Percona Server for MySQL in a docker container](https://docs.percona.com/percona-server/8.0/quickstart-docker.html) and [create a database and table in Percona Server for MySQL](https://docs.percona.com/percona-server/8.0/quickstart-database.html), you are ready to backup your databes with Percona XtraBackup.
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: To use Percona XtraBackup, you need to run it in its own Docker container and connect it to the Percona Server for MySQL container.

too many "it" - rewrite to be clear what "it" refers to

File: docs/quickstart-docker.md [hunk 21]
```diff
@@ -0,0 +1,143 @@
...
+    
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        backupvol
+        ```
+
+2. Start Percona XtraBackup 8.0.34 in a container, take and prepare a backup
+
+    To use the `docker run` command, specify the name or ID of the image you want to use and, optionally, some flags and arguments that modify the container's behavior.
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: Not clear what the "optionally" means - is this a description of the docker run command because we have several flags/arguments in the following table - so they are not optional? 
What's the difference between flags and arguments?

File: docs/quickstart-docker.md [hunk 22]
```diff
@@ -0,0 +1,143 @@
...
+    To use the `docker run` command, specify the name or ID of the image you want to use and, optionally, some flags and arguments that modify the container's behavior. 
+
+    The command has the following options:
+
+    | Option | Description |
+    |---|---|
+    | `--name`  |  Provides a name to the container. If you do not use this option, Docker adds a random name.  |
+    | `--volumes-from` | Refers to Percona Server for MySQL and indicates that you intend to use the same data as the `psmysql` container.|
+    | `it` | Interacts with the container and be a pseudo-terminal. |
+    | `--user root` | Sets the user to root inside the container. This option is required to access the MySQL data directory and run the xtrabackup command. |
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: which container?

File: docs/quickstart-docker.md [hunk 23]
```diff
@@ -0,0 +1,143 @@
...
+    |`backup_8034` |Indicates the directory inside the container where the backup files are stored. |
+    | `-v` | Mounts a volume from the host machine or another container to the container that is being run. |
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: more clarity. what does another container to the container mean?

File: docs/quickstart-docker.md [hunk 24]
```diff
@@ -0,0 +1,143 @@
...
+    | `backupvol` | Indicates the persistent storage for the database. |
+    | `psmysql`    | Indicates the Percona Server for MySQL container.  |
+    |`--password`   | Prompts the user to enter the password for the root user. For convenience you can add the password for the root user to this option, for example, `--password=secret`. Then the password will be passed automaticaly when running the command. Note that if you specify the password with `--password=secret`, the password is visible in `docker ps`, `docker ps -a` (docker history) and regular ps command.          |
+    | `8.0.34` | Use this tag to specify a specific version. |
+
+    When running Percona XtraBackup from a container and connecting to a Percona Server container, we recommend using the `–-user root` option in the Docker command.
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: Rewrite - we know that PXB is in a container and PS is in another container.

File: docs/quickstart-docker.md [hunk 25]
```diff
@@ -0,0 +1,143 @@
...
+
+    For this document, we are using the 8.0.34 tag. In Docker, a tag is a label assigned to an image. Tags are used to maintain different versions of an image. If we did not add a tag, Docker uses latest as the default tag and downloads the latest image from [percona/percona-xtrabackup on the Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup). The usage of tags also ensures that you are using compatible versions of Percona Server for MySQL and Percona XtraBackup.
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: `8.0.34` and `latest`

We haven't established that PXB only backups compatible releases. We stated earlier that it works with 8.0, but now we are saying something about compatible versions? 

We may consider adding a "Limitations" section to the Overview topic and refer to it from here.

File: docs/quickstart-docker.md [hunk 26]
```diff
@@ -0,0 +1,143 @@
...
+        Enter password:
+        ```
+    
+    In this example of expected output, we provide the first and last section of the backup log and use ellipses instead of the entire backup log.
+
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+
+        2023-12-14T15:45:16.277503-00:00 0 [Note] [MY-011825] [Xtrabackup] recognized server arguments: --datadir=/var/lib/mysql/
```
Comment (patrickbirch) [inline docs/quickstart-docker.md:65]: The example is too long. Unless you are walking the user through each of the entries, this doesn't make sense.

File: docs/quickstart-docker.md [hunk 27]
```diff
@@ -0,0 +1,143 @@
...
+
+    * You can check the Xtrabackup logs with the `docker container logs <container-name>` command.
+
+        For example:
+
+        ```{.bash data-prompt="$"}
+        $ docker container logs pxb
+        ```
+
+        After executing this command, you'll see the full xtrabackup log from `pxb` container.
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: This sentence is needed. You've already said you are checking the logs.

File: docs/quickstart-exit.md [hunk 28]
```diff
@@ -0,0 +1,91 @@
...
+* Exit the MySQL command client shell and the Docker container
+
+* Remove the Docker container and the Docker image
+
+* Remove the Docker volume.
+
+The steps are as follows:
+{.power-number}
+
+1. To exit the MySQL command client shell we use `exit`. You can also use the `\q` or `quit` commands. The execution of the statement also closes the connection.
```
Comment (patrickbirch) [inline docs/quickstart-exit.md]: Are we exiting the MySQL container or the PXB container? No idea.

File: docs/quickstart-exit.md [hunk 29]
```diff
@@ -0,0 +1,91 @@
...
+        ```{.text .no-copy}
+        Bye
+        ```
+
+2. Remove docker containers and images, if you no longer need docker containers / images or want to free up disk space. To remove a docker container, use the command `docker rm` followed by the container name or the container ID. To remove a docker image, use the command `docker rmi` followed by the image ID or name and the tag.
+
+    * Remove `psmysql2` and `pxb` Docker containers:
+
+        ```{.bash data-prompt="$"}
+        $ docker container rm psmysql2 -f
```
Comment (patrickbirch) [inline docs/quickstart-exit.md:31]: what does the `-f` do?

File: docs/quickstart-exit.md [hunk 30]
```diff
@@ -0,0 +1,91 @@
...
+        $ docker container rm pxb -f
+        ```
+
+        ??? example "Expected output"
+
+            ```{.text .no-copy}
+            pxb
+            ```
+
+    * Remove `percona/percona-xtrabackup:8.0.34` Docker image
```
Comment (patrickbirch) [inline docs/quickstart-exit.md]: Do i need to remove the psmysql image too?

File: docs/quickstart-overview.md (hunk 14 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-overview.md]: fixed

File: docs/quickstart-overview.md (hunk 15 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-overview.md]: fixed

File: docs/quickstart-overview.md (hunk 16 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-overview.md]: fixed

File: docs/quickstart-overview.md (hunk 18 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-overview.md]: added

File: docs/quickstart-overview.md (hunk 17 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-overview.md]: Updated

File: docs/quickstart-overview.md (hunk 19 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-overview.md]: added

File: docs/quickstart-docker.md (hunk 20 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: done

File: docs/quickstart-docker.md (hunk 22 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: added

File: docs/quickstart-docker.md (hunk 23 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: updated

File: docs/quickstart-docker.md (hunk 24 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: done

File: docs/quickstart-docker.md (hunk 25 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: Added a link

File: docs/quickstart-docker.md (hunk 26 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md:65]: done

File: docs/quickstart-overview.md [hunk 31]
```diff
@@ -1,35 +1,53 @@
-# Quickstart Guide for Percona XtraBackup
+# Overview
 
-Percona XtraBackup (PXB) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+Percona XtraBackup is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems.
```
Comment (patrickbirch) [inline docs/quickstart-overview.md:3]: Happy birthday.

File: docs/quickstart-docker.md (hunk 21 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: Updated

File: docs/quickstart-docker.md (hunk 27 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: removed

File: docs/quickstart-exit.md (hunk 28 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-exit.md]: Added

File: docs/quickstart-exit.md (hunk 29 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-exit.md:31]: added

File: docs/quickstart-exit.md (hunk 30 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-exit.md]: added

File: docs/quickstart-overview.md (hunk 31 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-overview.md:3]: Thank you!

**Review by patrickbirch — APPROVED** [review]
